cross-chain-atomic-swap-cordapp/src/main/kotlin/jp/co/layerx/cordage/crosschainatomicswap/Utils.kt

```diff
@@ -30,11 +30,11 @@ fun Party.ethAddress(): String {
             ethAddress.load(url.openStream())
         }
         val root = System.getProperty("user.dir")
-        val path = Paths.get("$root/cordapps/config/cross-chain-atomic-swap-cordapp-0.1.conf")
+        val path = Paths.get("$root/cordapps/config/config.conf")
```

Comment:
I am wondering if this works with flows in the deployed nodes because the name of the config file will be changed when you deploy nodes.

Reply:
Yeah it was a mistake just changed it ^^